### PR TITLE
RDFStart e2e tests added

### PR DESCRIPTION
**Pull request by teodossidossev**
fixed e2e tests graphdb.js dependency

### CODE REVIEW

File: test-e2e/package.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
     "axios": "^0.18.1",
     "express": "^4.17.1",
     "form-data": "^2.3.3",
-    "graphdb": "^1.0.0",
+    "graphdb": "github:Ontotext-AD/graphdb.js",
```
Comment (svilenvelikov) [inline test-e2e/package.json:18]: We should not forget this one.

File: test-e2e/tests/data/rdfstar/expected_response_sparql_tsv_star.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+?s
```
Comment (svilenvelikov) [inline test-e2e/tests/data/rdfstar/expected_response_sparql_tsv_star.txt:1]: what is this `?s`

Comment (teodossidossev) [reply test-e2e/tests/data/rdfstar/expected_response_sparql_tsv_star.txt:1]: A sparql tsv (tab separated values) response in string format

File: test-e2e/tests/rdfstar.spec.js [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+    let payload = new GetQueryPayload()
+      .setQuery('describe <<<http://www.wikidata.org/entity/Q472> <http://www.wikidata.org/prop/direct/P1889> <http://www.wikidata.org/entity/Q202904>>>')
+      .setQueryType(QueryType.DESCRIBE)
+      .setResponseType(RDFMimeType.TURTLE_STAR)
+      .setLimit(100);
+
+    rdfClient.registerParser((new RDFXmlParser()));
+    return rdfClient.query(payload).then((resp) => {
+      resp.on('data', data => {
+        expect(JSON.stringify(data.toString())).toEqual((JSON.stringify(expected)));
```
Comment (desislava-hristova-ontotext) [inline test-e2e/tests/rdfstar.spec.js:39]: Why do we need JSON.stringify, since it is not JSON actually?

Comment (teodossidossev) [reply test-e2e/tests/rdfstar.spec.js]: fixed

**Pull request merged**